noir-projects/noir-contracts/contracts/token_blacklist_contract/src/types/roles.nr

```diff
@@ -10,19 +10,8 @@ pub struct UserFlags {
     is_blacklisted: bool,
 }
 
-impl FromField for UserFlags {
```

Comment:
SharedMutable now requires T to implement `Packable` and not `FromField` and `ToField`